source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluationCalifications.cs

```diff
@@ -141,11 +141,7 @@ private CalificationsDto GetFinishedEvaluation(CalificationsEvaluationDto evalua
         {
             return new CalificationsDto()
             {
-                View = (_loggedUser == evaluationDto.ResponsibleId || _isEmployeeManager)
-                ? UserView.Company
-                    : (_loggedUser == _evaluatedUser)
-                        ? UserView.Auto
-                        : UserView.Evaluation,
+                View = UserView.Company,
```

Comment:
Well, I thought that `UserView` was represented the view that we allowed the user to see  depending on the evaluation state and the user itself, but your perspective is also valid.
Although I agree that changing the `UserView` is a risky move since it has a lot of references to check how the page is going to behave.
I'll look for another alternative